doc/Language/objects.pod6

```diff
@@ -138,7 +135,7 @@ another class.
 =head2 Attributes
 X<|Attribute> X<|Property> X<|Member> X<|Slot>
 
-Attributes are variables that exist per instance of a class. They are where
+Attributes are variables that exist per object basis. They are where
```

Comment:
I like that except classes are objects, so that's a little weird.

Attributes are variables that exist per instance of a class.

Maybe that?